main.go

```diff
@@ -220,40 +220,36 @@ func apply(configFile string) (ctrl.Options, config.Configuration) {
 		Scheme: scheme,
 	}
 
-	config := config.Configuration{}
+	cfg := config.Configuration{}
 
 	if configFile == "" {
-		scheme.Default(&config)
+		scheme.Default(&cfg)
+		options, err = options.AndFrom(&cfg)
+		if err != nil {
+			setupLog.Error(err, "unable to load the default config")
+			os.Exit(1)
+		}
+
+		cfgStr, err := encodeConfig(&cfg)
+		if err != nil {
+			setupLog.Error(err, "unable to encode the default config")
+			os.Exit(1)
+		}
+		setupLog.Info("Successfully loaded the default config", "config", cfgStr)
```

Comment:
This logger output the below:

```shell
{"level":"info","ts":"2022-10-12T16:30:30.00241243Z","logger":"setup","caller":"logr@v1.2.3/logr.go:261","msg":"Successfully loaded the default config","config":"apiVersion: config.kueue.x-k8s.io/v1alpha2\nhealth:\n  healthProbeBindAddress: :8081\ninternalCertManagement:\n  enable: true\n  webhookSecretName: kueue-webhook-server-cert\n  webhookServiceName: kueue-webhook-service\nkind: Configuration\nmanageJobsWithoutQueueName: false\nmetrics:\n  bindAddress: :8080\nnamespace: kueue-system\nwebhook:\n  port: 9443\n"}
```

Reply:
we could de-duplicate this code by putting it outside of the conditional.

A message such as `Successfully loaded configuration` would cover both codepaths.

Reply:
Thanks for the suggestion!
Done.